<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-01T01:29:24 IST" name="Suite" finished-at="2024-08-01T01:29:40 IST" duration-ms="16792">
    <groups>
    </groups>
    <test started-at="2024-08-01T01:29:24 IST" name="Test" finished-at="2024-08-01T01:29:40 IST" duration-ms="16792">
      <class name="seleniumjava.OrangeHRM">
        <test-method is-config="true" signature="Setup()[pri:0, instance:seleniumjava.OrangeHRM@5656be13]" started-at="2024-08-01T01:29:24 IST" name="Setup" finished-at="2024-08-01T01:29:39 IST" duration-ms="14818" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method signature="testLogo()[pri:1, instance:seleniumjava.OrangeHRM@5656be13]" started-at="2024-08-01T01:29:39 IST" name="testLogo" finished-at="2024-08-01T01:29:39 IST" duration-ms="85" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogo -->
        <test-method signature="testAppUrl()[pri:2, instance:seleniumjava.OrangeHRM@5656be13]" started-at="2024-08-01T01:29:39 IST" name="testAppUrl" finished-at="2024-08-01T01:29:39 IST" duration-ms="36" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://opensource-demo.orangehrmlive.com/web/index.pip/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.pip/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at seleniumjava.OrangeHRM.testAppUrl(OrangeHRM.java:42)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAppUrl -->
        <test-method signature="testHomepageTitle()[pri:3, instance:seleniumjava.OrangeHRM@5656be13]" started-at="2024-08-01T01:29:39 IST" name="testHomepageTitle" finished-at="2024-08-01T01:29:39 IST" depends-on-methods="seleniumjava.OrangeHRM.testAppUrl" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method seleniumjava.OrangeHRM.testHomepageTitle() on instance seleniumjava.OrangeHRM@5656be13 depends on not successfully finished methods 
[seleniumjava.OrangeHRM.testAppUrl() on instance seleniumjava.OrangeHRM@5656be13]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method seleniumjava.OrangeHRM.testHomepageTitle() on instance seleniumjava.OrangeHRM@5656be13 depends on not successfully finished methods 
[seleniumjava.OrangeHRM.testAppUrl() on instance seleniumjava.OrangeHRM@5656be13]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testHomepageTitle -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:seleniumjava.OrangeHRM@5656be13]" started-at="2024-08-01T01:29:39 IST" name="teardown" finished-at="2024-08-01T01:29:40 IST" duration-ms="978" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- seleniumjava.OrangeHRM -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
